pdstemplate/__init__.py

```
@@ -1125,6 +1129,15 @@ def FILE_TIME(filepath):
             `filepath` in the form "yyyy-mm-ddThh:mm:ss".
         """
 
+        ##################################################################################
+        local_filepath = FCPath(filepath).retrieve() ## this will just return the last retrieved
+                                                     ## time, but it's better than a crash
+        if local_filepath != filepath:
+            filepath = local_filepath
+            logger = get_logger()
+            logger.warning(f'FILE_TIME reflects the cached time: {filepath}', force=True)
```

Ruff (0.14.8)

Logging statement uses f-string

(G004)